OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -97,6 +97,9 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		[Desc("Does this actor cancel its previous activity after resupplying?")]
 		public readonly bool AbortOnResupply = true;
 
+		[Desc("Can this actor be given an explicit land order using the force-move modifier?")]
+		public readonly bool EnableForceLanding = true;
```

Comment:
If the answer to https://github.com/OpenRA/OpenRA/pull/16768#discussion_r303788332 is "no" then this could/should be called `CanForceLand` to match. Otherwise I wonder whether `AllowForceLand` would be better here?

Reply:
Renamed.